packages/selenium/test/RunPartial/UseLegacyTests.cs

```diff
@@ -0,0 +1,51 @@
+using System.Linq;
+using Deque.AxeCore.Selenium;
+using NUnit.Framework;
+
+namespace Deque.AxeCore.Selenium.Test.RunPartial
+{
+    public class UseLegacyTests : TestBase
+    {
+        [Test]
+        public void ShouldRunLegacyAxe()
+        {
+            GoToFixture("index.html");
+
+            var results = new AxeBuilder(driver, CustomSource($"{axeSource}{axeRunPartialThrows}"))
+                .UseLegacyMode(true)
```

Comment:
These `UseLegacyMode` invocations should be wrapped in `#pragma warning disable CS0618` / `#pragma warning enable CS0618` directives to avoid the build warnings, since the references to obsolete APIs are intentional/correct